include/clap/ext/draft/undo.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "../../plugin.h"
+#include "../../string-sizes.h"
+
+static CLAP_CONSTEXPR const char CLAP_EXT_UNDO[] = "clap.undo.draft/0";
+
+// Describes an undo step
+typedef struct clap_undo_info {
+   // This is the unique identifier of this undo step.
+   // It is valid until loading a state or destroying the plugin.
```

Comment:
also still valid when after_destructive_change gets called? Maybe that should be mentioned here as well if relevant

Reply:
Newly produced id shouldn't clash with those produced before`after_destructive_change()`.
I'd have one `clap_id next_id` that I'd only increment for the whole lifetime of the plugin instance.